website/project/model.py

```diff
@@ -890,6 +882,8 @@ def institution_relationship_url(self):
 
     def __init__(self, *args, **kwargs):
 
+        kwargs.pop('logs', [])
```

Comment:
You're correct, the ticket says to leave the data there, and modularodm automatically inserts it when loading up an old node. There's probably a way to specify not to, but that's not a rabbit hole worth going down at the moment.